### PR TITLE
#FN-33: User should be able to Chat

### PR DESCRIPTION
**Pull request by Niyobuzima**
# PR Description
Trello: https://trello.com/c/xglKKsHz
Having travellers interacting with each other on the platform adds value to it as it allows them to asks questions about places, offer recommendations, guidance etc...

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] User should open the chat tab
- [x] User should see past messages
- [x] User should be able to send messages in public chat

## Description of tasks that were expected to be completed and that are not yet completed

- [x] none

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `yarn i` to install all required dependencies
2. run `yarn run dev`

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/redux/store/store.tsx [hunk 1]
```diff
@@ -30,12 +31,9 @@ const store = configureStore({
     Allproducts: AllProductReducer,
     viewProduct: fetchViewReducer,
     updateProduct: updateProductReducer,
+    chat: chatReducer,
   },
-  middleware: getDefaultMiddleware({
-    serializableCheck: false,
```
Comment (Illustre13) [inline src/redux/store/store.tsx:35]: You can add this back for handling the redux serialization data parsing !!

Comment (Illustre13) [inline src/redux/store/store.tsx:50]: You can add this back for handling the redux serialization data parsing !!

**Review by Illustre13 — CHANGES_REQUESTED** [review]
You can add back for handling the redux serialization data parsing !!

**Review by Illustre13 — APPROVED** [review]

**Review by Honore123 — APPROVED** [review]

**Review by EvelyneUmubyeyi — APPROVED** [review]

**Review by dushimeemma — COMMENTED** [review]
While do you request me to enter my name when I am authenticated @Niyobuzima?

**Review by dushimeemma — APPROVED** [review]

**Pull request merged**